serving/samples/helloworld-scala/helloworld-scala.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: serving.knative.dev/v1alpha1
+kind: Service
+metadata:
+  name: helloworld-scala
+  namespace: default
+spec:
+  runLatest:
+    configuration:
+      revisionTemplate:
+        spec:
+          container:
+            image: "your_repository_name/helloworld-scala:0.0.1"
+            imagePullPolicy: IfNotPresent
+            env:
+              - name: MESSAGE
+                value: "Scala & Akka on Knative says hello!"
+              - name: HOST
+                value: "localhost"
+              - name: PORT
+                value: "8080"
```

Comment:
Knative [will set the `$PORT` environment variable](https://github.com/knative/serving/blob/master/docs/runtime-contract.md#process) (default 8080) to the chosen serving port, so this declaration won't do anything.